dotenv/mod_test.ts

```diff
@@ -29,6 +29,18 @@ Deno.test("load() handles non-existent .env files", async () => {
   assertEquals({}, loadSync(loadOptions));
 });
 
+Deno.test("load() handles URL as path for .env files", async () => {
+  const conf = loadSync({
+    envPath: new URL(path.join(testdataDir, ".env"), import.meta.url),
```

Comment:
`new URL(path.join(testdataDir, ".env"), import.meta.url)`

This doesn't work on windows as the paths are `\` separated on windows, but that doesn't work as URL string

You need to call `toFileUrl` from `@std/path`

Reply:
Do you mean like this:

```ts
new URL(
  toFileUrl(path.join(testdataDir, ".env")),
  import.meta.url,
)
```
?